src/main/kotlin/com/team4099/robot2022/commands/led/ChangeLedState.kt

```diff
@@ -0,0 +1,59 @@
+package com.team4099.robot2022.commands.led
+
+import com.team4099.robot2022.config.Constants
+import com.team4099.robot2022.subsystems.LED
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class ChangeLedState(val changestate: Constants.LED.LEDState) : CommandBase() {
+  /*
+  init {
+    addRequirement(Feeder)
+    addRequirement(Intake)
+    addRequirement(Shooter)
+    addRequirement(LED)
+  }
+
+  override fun initialize() {
+    val isChanging = false
+
+    if (Intake.intakeState == 1.0 || Shooter.shooterState == 1200.0) {
+      val isChanging = true
+    }
+    when (Feeder.ballCount) {
```

Comment:
instead of doing feeder ball count, look at intake motor current draw and if exceeds a certain value increase ball count